frontend/public/locales/fr/protected-profile.json

```diff
@@ -0,0 +1,10 @@
+{
+  "applicant-information": {
```

Comment:
French content is available on [Figma](https://www.figma.com/design/d8XksFd7eDtBF9V3IvaBeb/CDCP-Client-Portal--in-MSCA-?node-id=9217-8214&t=JTPNZXdMIgFf2Qkc-0)